components/frontend/src/dashboard/StatusBarChart.js

```diff
@@ -8,7 +8,9 @@ function nr_metrics_label(nr_metrics) {
 
 export function StatusBarChart({ animate, colors, label, tooltip, summary, maxY, style }) {
     const nrMetrics = sum(summary[Object.keys(summary)[0]])
-    const barRatio = Math.max(5, nrMetrics) / maxY
+    const nrDates = Object.keys(summary).length
+    const horizontalPadding = 50 + (7 - nrDates) * 10
```

Comment:
Perhaps a comment to describe the calculation would help, seems like magic numbers to me

Reply:
Code changed, but explanation added for the new calculation.